sdk/ml/azure-ai-ml/azure/ai/ml/operations/_evaluator_operations.py

```diff
@@ -0,0 +1,338 @@
+        name: str,
+        stage: Optional[str] = None,
+        *,
+        list_view_type: ListViewType = ListViewType.ACTIVE_ONLY,
+    ) -> Iterable[Model]:
+        """List all model assets in workspace.
+
+        :param name: Name of the model.
+        :type name: str
+        :param stage: The Model stage
+        :type stage: Optional[str]
+        :keyword list_view_type: View type for including/excluding (for example) archived models.
+            Defaults to :attr:`ListViewType.ACTIVE_ONLY`.
+        :paramtype list_view_type: ListViewType
+        :return: An iterator like instance of Model objects
+        :rtype: ~azure.core.paging.ItemPaged[~azure.ai.ml.entities.Model]
+        """
+        properties_str = "is-promptflow=true,is-evaluator=true"
+        if name:
+            return cast(
+                Iterable[Model],
+                (
+                    self._model_op._model_versions_operation.list(
+                        name=name,
+                        registry_name=self._model_op._registry_name,
+                        cls=lambda objs: [Model._from_rest_object(obj) for obj in objs],
+                        properties=properties_str,
+                        **self._model_op._scope_kwargs,
+                    )
+                    if self._registry_name
+                    else self._model_op._model_versions_operation.list(
+                        name=name,
+                        workspace_name=self._model_op._workspace_name,
+                        cls=lambda objs: [Model._from_rest_object(obj) for obj in objs],
+                        list_view_type=list_view_type,
+                        properties=properties_str,
+                        stage=stage,
+                        **self._model_op._scope_kwargs,
+                    )
+                ),
+            )
+        # ModelContainer object does not carry properties.
+        raise UnsupportedOperationError("list on evaluation operations without name provided")
+        # TODO: Implement filtering of the ModelContainerOperations list output
+        # return cast(
+        #     Iterable[Model], (
+        #         self._model_container_operation.list(
+        #             registry_name=self._registry_name,
+        #             cls=lambda objs: [Model._from_container_rest_object(obj) for obj in objs],
+        #             list_view_type=list_view_type,
+        #             **self._scope_kwargs,
+        #         )
+        #         if self._registry_name
+        #         else self._model_container_operation.list(
+        #             workspace_name=self._workspace_name,
+        #             cls=lambda objs: [Model._from_container_rest_object(obj) for obj in objs],
+        #             list_view_type=list_view_type,
+        #             **self._scope_kwargs,
+        #         )
+        #     )
+        # )
+
+    @monitor_with_activity(ops_logger, "Evaluator.Share", ActivityType.PUBLICAPI)
+    @experimental
```

Comment:
I think this is also not necessary if you tag on the operation group property,

Reply:
Fixed